e2e/command_test.go

```diff
@@ -40,7 +38,16 @@ type cmdBuilder struct {
 
 // getEnvoy returns a new command builder.
 func getEnvoy(arg0 string) *cmdBuilder { //nolint:golint
-	return &cmdBuilder{exec.Command(getEnvoyPath, arg0)} //nolint:gosec
+
+	var args []string
+	if arg0 == "--version" {
+		args = []string{arg0}
+	} else {
+		args = []string{"--internal-user-agent", userAgent, arg0}
```

Comment:
is it intentional to have "--internal-user-agent" before arg0?

Reply:
yeah it is a global flag

Reply:
I could rename arg0 to "command" but #246 will drift this anyway :D

Reply:
I mean, if it is a global flag, it should work even put this after arg0?

Reply:
nope.. urfave is neater. global flags are before the command. This is helpful also because you know nothing will ever interfere with args if a subcommand defines no flags

```
USAGE:
   getenvoy [global options] command [command options] [arguments...]
```

Reply:
neat!